admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     geolocation: './js/pages/geolocation',
     imports: './js/pages/import',
     improve_design_positions: './js/pages/improve/design_positions',
+    improve_design_positions_module_hook_form: './js/pages/improve/design_positions/module-hook-form.js',
```

Comment:
Do you think it could be shortened?

Reply:
why? ^^

Reply:
Are we starting to follow "side menu" structure for naming files as well? :thinking: 

PS architecture will be based on side menu, hmmm.